app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    current_user = User.find(params[:user_id])
```

Comment:
- [ ] @AlselwiAisha Kindly consider adding the current_user method in your application controller and then use it here, instead of creating a variable as per the project instructions
![image](https://github.com/AlselwiAisha/BlogApp/assets/61408860/c899303b-686b-4c06-a3de-54ff168a8a97)